src/OpenTelemetry.SemanticConventions/Attributes/AndroidAttributes.cs

```diff
@@ -15,35 +15,35 @@ namespace OpenTelemetry.SemanticConventions;
 public static class AndroidAttributes
 {
     /// <summary>
-    /// Uniquely identifies the framework API revision offered by a version (<c>os.version</c>) of the android operating system. More information can be found <a href="https://developer.android.com/guide/topics/manifest/uses-sdk-element#ApiLevels">here</a>.
+    /// Uniquely identifies the framework API revision offered by a version (<c>os.version</c>) of the android operating system. More information can be found <a href="https://developer.android.com/guide/topics/manifest/uses-sdk-element#ApiLevels">here</a>
```

Comment:
Can we keep `.` at the end of documentation comment? Typically it leads to https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1629.md

Reply:
Yes, there's already a issue in Weaver to add support to adding dots at the end. https://github.com/open-telemetry/weaver/issues/353